addons/mass_mailing/models/mailing_list.py

```diff
@@ -38,7 +38,7 @@ class MassMailingList(models.Model):
         string='Subscription Information',
         copy=True, depends=['contact_ids'])
     is_public = fields.Boolean(
-        string='Show In Preferences', default=True,
+        string='Show In Preferences', default=False,
         help='The mailing list can be accessible by recipients in the subscription '
              'management page to allows them to update their preferences.')
```

Comment:
While we're at it, can we modify this help to remove the "s" of "to allows" ?

You will have to export the POT file as well to add the new entry.

Reply:
Hello, @awa-odoo
I have made the changes, can you please have a look?

Thanks :slightly_smiling_face: